docs/source/custom_flows.rst

```diff
@@ -170,6 +170,8 @@ GHDL
 * Extend the ``ghdl -r`` call with the option
   ``--vpi=$(cocotb-config --lib-name-path vpi ghdl)``.
 
+* VHDL architectures can be specified via the `$(ARCH)` make variable
+
```

Comment:
This should go in `simulator_support.rst` under the GHDL section.

Reply:
ok, moving it there.